tests/test_ingestion.py

```diff
@@ -184,6 +178,16 @@ def testFindMatchingFiles(self):
 
 def _isEmpty(butler, datasetType):
     """Test that a butler repository contains no objects.
+
+    Parameters
+    ----------
+    datasetType : `str`
+        The type of dataset to search for.
+
+    Notes
+    -----
+    .. warning::
+       Does not work for calib datasets, because they're not discoverable.
```

Comment:
Good note. Does this deserve a ticket and TODO?

Reply:
See discussion on [#dm](https://lsstc.slack.com/archives/C2JPL2DGD/p1525816663000392). To me, it felt like a "that's not a bug, that's a feature" moment.